hack/gen-api-docs.sh

```diff
@@ -44,7 +44,7 @@ mkdir -p "${genSrc}"
 pushd "${genSrc}"
 
 gen_ref_docs "application-api" "Application"
-gen_ref_docs "enterprise-contract" "Enterprise Contract" "https://github.com/enterprise-contract/enterprise-contract-controller.git"
+gen_ref_docs "conforma" "Conforma" "https://github.com/enterprise-contract/enterprise-contract-controller.git"
```

Comment:
Do we keep the git URL as it is (`enterprise-contract`) or will it be update to `conforma` as well ?

Reply:
It's a good question. It will be updated, but not yet. (We're migrating repos one at time. This one is tracked in [EC-1112](https://issues.redhat.com/browse/EC-1112), and still to do.)